packages/react-i18n/src/test/i18n.test.ts

```diff
@@ -616,7 +616,7 @@ describe('I18n', () => {
       });
       const expected = new Intl.DateTimeFormat(defaultDetails.locale, {
         timeZone: 'UTC',
-      }).format(date);
+      }).format(new Date('2018-01-01'));
```

Comment:
Why this

Reply:
This was an existing bug that only seemed to come up when I was coding in the wee hours of the morning. I modified this code to reflect the test above it.